pxteditor/editorcontroller.ts

```diff
@@ -77,6 +76,7 @@ export function bindEditorMessages(getEditorAsync: () => Promise<IProjectView>)
                             case "closeflyout": return Promise.resolve().then(() => projectView.closeFlyout());
                             case "unloadproject": return Promise.resolve().then(() => projectView.unloadProjectAsync());
                             case "saveproject": return projectView.saveProjectAsync();
+                            case "compile": return projectView.compile();
```

Comment:
please add this message in two places:
* the typing for `EditorMessageRequest` in `localtypings/pxteditor.d.ts`.
* as a method in the `EditorDriver` class in `pxtservices/editorDriver.ts`.

i'll open a pr that fixes this switch statement to disallow strings outside of the typing; looks like the call to toLowerCase() broke the type inference.